### PR TITLE
Jk styling

### PR DESCRIPTION
**Pull request by joekennerly**
add style to news

### CODE REVIEW

File: career.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="main.css" />
-    <title>Document</title>
+    <title>Will Smith | Career</title>
```
Comment (AdamKnowles) [inline career.html:8]: Nice!

**Comment by AdamKnowles** [comment]
👍

**Comment by dhobson21** [comment]
This looks pretty dang good, IMHO. 👍

**Comment by joekennerly** [comment]
joy?

**Comment by bestyoucanbe** [comment]
Wondered why there were 4 commits but you explained it well (verbally).  Thanks!  Looked at your code.  Keep up the good work!

**Comment by bestyoucanbe** [comment]
👍

**Comment by joekennerly** [comment]
Ty for the feed back...
Go Krabs!

**Pull request merged**